### PR TITLE
make nut06 fields optional

### PR DESCRIPTION
**Pull request by thesimplekid**

### CODE REVIEW

File: 06.md [hunk 1]
```diff
@@ -81,14 +81,14 @@ With the mint's response being of the form `GetInfoResponse`:
...
-- `name` is the name of the mint and should be recognizable. 
-- `pubkey` is the hex pubkey of the mint.
-- `version` is the implementation name and the version of the software running on this mint separated with a slash "/", 
-- `description` is a short description of the mint that can be shown in the wallet next to the mint's name. 
-- `description_long` is a long description that can be shown in an additional field.
-- `contact` is an array of contact objects to reach the mint operator. A contact object consists of two fields. The `method` field denotes the contact method (like "email"), the `info` field denotes the identifier (like "contact@me.com").
-- `modt` is the message of the day that the wallet must display to the user. It should only be used to display important announcements to users, such as scheduled maintenances. 
-- `nuts` indicates each NUT specification that the mint supports and its settings. The settings are defined in each NUT separately. 
+- (optional) `name` is the name of the mint and should be recognizable. 
+- (optional) `pubkey` is the hex pubkey of the mint.
```
Comment (prusnak) [inline 06.md:85]: isn't at least pubkey mandatory?

File: 06.md [hunk 2]
```diff
@@ -81,14 +81,14 @@ With the mint's response being of the form `GetInfoResponse`:
...
 
-- `name` is the name of the mint and should be recognizable. 
-- `pubkey` is the hex pubkey of the mint.
-- `version` is the implementation name and the version of the software running on this mint separated with a slash "/", 
-- `description` is a short description of the mint that can be shown in the wallet next to the mint's name. 
-- `description_long` is a long description that can be shown in an additional field.
-- `contact` is an array of contact objects to reach the mint operator. A contact object consists of two fields. The `method` field denotes the contact method (like "email"), the `info` field denotes the identifier (like "contact@me.com").
-- `modt` is the message of the day that the wallet must display to the user. It should only be used to display important announcements to users, such as scheduled maintenances. 
-- `nuts` indicates each NUT specification that the mint supports and its settings. The settings are defined in each NUT separately. 
+- (optional) `name` is the name of the mint and should be recognizable.
```
Comment (prusnak) [inline 06.md:84]: maybe we should make the `name` mandatory too?

File: 06.md (hunk 1 above)
Comment (callebtc) [reply 06.md:85]: Nope, this pubkey isn't used for anything productive yet. The relevant ecash pubkeys are found in `/v1/keys`

**Review by callebtc — APPROVED** [review]
LGTM

Good strategy to make everything optional now and later decide what should be mandatory.

**Review by prusnak — APPROVED** [review]

File: 06.md (hunk 2 above)
Comment (thesimplekid) [reply 06.md]: I think at the spec level we should leave it optional so its not relied on for serialization but implementation should strongly encourage setting one.

**Pull request merged**